WordPress/WordPressTest/MediaImageServiceTests.swift

```diff
@@ -97,7 +97,8 @@ class MediaImageServiceTests: CoreDataTestCase {
         media.width = 1024
         media.height = 680
         media.remoteThumbnailURL = "https://example.files.wordpress.com/2023/09/video-thumbnail.jpg"
-        try mainContext.obtainPermanentIDs(for: [media])
+        media.blog = makeEmptyBlog()
+        try mainContext.save()
```

Comment:
This has nothing to do with SwiftLint. It's an attempt to address a test failure saw in CI and that I experienced locally, too. See 26ffee6087e5b27998902745fc164c234c7067c6